ddp-workspace/projects/ddp-dsm-ui/src/app/scan/scan.component.ts

```diff
@@ -193,10 +195,11 @@ export class ScanComponent implements OnInit {
         if (this.scanTracking) {
           const scanPayloads = [];
           this.scanPairsValue.forEach(element => {
+            if(element.rightValue !== '' && element.leftValue !== '') {
             scanPayloads.push({
-              kitLabel: element.rightValue,
-              trackingReturnId: element.leftValue
-            });
+              kitLabel: element.leftValue,
+              trackingReturnId: element.rightValue
```

Comment:
did we change the order of things here?

Reply:
Thank you for pointing this out! I thought we did not, but we did. I reverted this specific change.